README.md

```diff
@@ -75,14 +75,17 @@ create isolated Python environment is recommended for this project.
 #### Testing
 6. Run all unit tests
    ```python
-   cd ml-conversational-analytic-tool
    python -m unittest discover -s tests
    ```
 7. Run an individual unit test
    ```python
-   cd ml-conversational-analytic-tool
    python -m unittest tests/<file_name>
    ```
+8. By using tox
```

Comment:
nit: I don't think the testing section needs to be numbered as it indicates steps instead of alternatives. could we use bullet points instead?

Reply:
Done.